spec/unit/outputs/kafka_spec.rb

```diff
@@ -103,7 +113,7 @@
   end
 
   context "when KafkaProducer#send() raises a non-retriable exception" do
-    let(:failcount) { (rand * 10).to_i }
+    let(:failcount) { (rand * 10 + 1).to_i }
```

Comment:
not sure if this is a relative change

Reply:
It is not related to the goal of the changeset but I got annoyed that the first patchset uploaded failed due to the instability of this test case, I can revert the change if needed.

Reply:
this test case started failing again after I reverted the change, should I re-introduce it?